pootle/apps/pootle_store/models.py

```diff
@@ -1351,11 +1331,6 @@ def get_last_updated_info(self):
 class StoreManager(models.Manager):
```

Comment:
> It seems that now there is not a big difference between this and simple_objects, so perhaps simple_objects can be dropped as well.

probably - but that would require changing anywhere that currently uses - so i reckon that is a separate PR